packages/shared/src/modules/users/libs/enums/user-validation-message.enum.ts

```diff
@@ -13,8 +13,9 @@ const UserValidationMessage = {
 	EMAIL_REQUIRE: "Email is required",
 	EMAIL_TAKEN: "Email is taken",
```

Comment:
According to requirements message for EMAIL_TAKEN shold be "This email is already registered. Please use a different email"